.github/workflows/build-yocto.yml

```diff
@@ -7,6 +7,9 @@ on:
     branches:
       - main
 
+env:
+  PERSISTENT_DIR: /srv/gh-runners/quic-yocto
```

Comment:
In the beginning I have used the same PERSISTENT_DIR but this will also affect bitbake and was the reason of my change from PERSISTENT_DIR -> CACHE_DIR as https://github.com/quic-yocto/meta-qcom-hwe/pull/41#issuecomment-2419994345

Reply:
https://docs.yoctoproject.org/bitbake/2.10/bitbake-user-manual/bitbake-user-manual-ref-variables.html#term-PERSISTENT_DIR

Reply:
right. Let me revert to CACHE_DIR.